opm/autodiff/StandardWell.hpp

```diff
@@ -25,7 +25,7 @@
 
 
 #include <opm/autodiff/WellInterface.hpp>
-#include <opm/autodiff/ISTLSolver.hpp>
```

Comment:
stupid question: if this is the only change required, is ISTLSolverEbos used in this file at all?